keps/sig-auth/3221-structured-authorization-configuration/README.md

```diff
@@ -581,17 +624,15 @@ the scenarios.
 
 ### Upgrade / Downgrade Strategy
 
-While the feature is in Alpha, there is no change if cluster administrators want to
-keep on using command line flags.
+There is no change if cluster administrators want to keep on using command line flags.
 
-When the feature goes to Beta/GA or the cluster administrators want to configure
-authorizers using a config file, they need to make sure the config file exists before
-upgrading the cluster. Similarly when downgrading clusters, they would need to add
-the flags back to their bootstrap mechanism.
+If the cluster administrators wants to configure authorizers using a config file,
```

Comment:
Do we need to mention anything about multiple apiservers and what happens when theres a mismatch between config file and flag?

Reply:
We can, though it's not really different than rolling out an authorization config change across multiple servers using solely flags.

Reply:
added a couple lines about this